sort.java

```diff
@@ -21,7 +21,7 @@ void display(int arr[]) {
 
     public static void main(String[] args) {
         Sort ob = new Sort();
-        int arr[] = {6, 4, 5, 12, 2, 11, 9};
+        int arr[] = {18, 14, 5, 12, 2, 11, 9};
```

Comment:
This object also can have a better name instead of `arr`. The idea is for your team-mates/reviewers to understand what this object is for by simply looking at the name.